applications/WindEngineeringApplication/python_scripts/set_mesh_motion_and_get_forces_process.py

```diff
@@ -0,0 +1,261 @@
+import KratosMultiphysics
+import KratosMultiphysics.MeshMovingApplication as MeshMoving
+from KratosMultiphysics.time_based_ascii_file_writer_utility import TimeBasedAsciiFileWriterUtility
+
+import math
+import sys
+
+
+
+def Factory(parameters, model):
+    if not isinstance(parameters, KratosMultiphysics.Parameters):
+        raise TypeError("expecting a Parameters object encapsulating a json string, but got {}".format(type(parameters)))
+    return SetMeshMotionAndGetForcesProcess(model, parameters["Parameters"])
+
+
+
+class SetMeshMotionAndGetForcesProcess(KratosMultiphysics.Process):
+    def __init__(self, model, parameters):
+        KratosMultiphysics.Process.__init__(self)
+
+        self.parameters = parameters
+        self.parameters.ValidateAndAssignDefaults(self.GetDefaultParameters())
+
+        if self.parameters['interval'][1].IsString():
+            if self.parameters['interval'][1].GetString() == "End":
+                self.parameters['interval'][1].SetDouble(1e30)
+            else:
+                raise Exception("The second value of interval can be 'End' or a number, interval currently:" +
```

Comment:
better use the `IntervalUtility` here

Reply:
Ah I could have sworn I already converted all of these to ```IntervalUtility``` but I guess I was wrong, sorry.